apps/core/lib/core/schema/provider_scaffold.ex

```diff
@@ -18,6 +18,9 @@ defmodule Core.Schema.ProviderScaffold do
 
   defp provider_file(name, vsn) do
     Path.join([:code.priv_dir(:core), "scaffolds/terraform/providers/#{vsn}", "#{name}.eex"])
-    |> File.read!
+    |> File.read
+    |> case do
```

Comment:
You need to also handle the `{:ok, result}` case, probably with a case clause like:

```elixir
{:ok, contents} -> contents
```